lang/cs/Org.Apache.REEF.IMRU/Org.Apache.REEF.IMRU.csproj

```diff
@@ -74,6 +76,9 @@ under the License.
     <Compile Include="OnREEF\Driver\DataLoadingContext.cs" />
     <Compile Include="OnREEF\Driver\EvaluatorManager.cs" />
     <Compile Include="OnREEF\Driver\EvaluatorSpecification.cs" />
+    <Compile Include="OnREEF\Driver\IJobLifeCycleManager.cs" />
```

Comment:
we have default implementation, but for extensibility I think we should allow client to provide their implementation if needed

---
In reply to: [86018616](https://github.com/apache/reef/pull/1161#discussion_r86018616) [](ancestors = 86018616)